lib/internal/url.js

```diff
@@ -65,12 +65,9 @@ function toUSVString(val) {
 // Refs: https://html.spec.whatwg.org/multipage/browsers.html#concept-origin-opaque
 const kOpaqueOrigin = 'null';
 
-// Refs:
-// - https://html.spec.whatwg.org/multipage/browsers.html#unicode-serialisation-of-an-origin
-// - https://html.spec.whatwg.org/multipage/browsers.html#ascii-serialisation-of-an-origin
-function serializeTupleOrigin(scheme, host, port, unicode = true) {
-  const unicodeHost = unicode ? domainToUnicode(host) : host;
-  return `${scheme}//${unicodeHost}${port === null ? '' : `:${port}`}`;
+// Refs: https://html.spec.whatwg.org/multipage/browsers.html#ascii-serialisation-of-an-origin
+function serializeTupleOrigin(scheme, host, port) {
```

Comment:
Is there a need to? This function is purely internal, and there is no remnant from the previous Unicode serialization that needs explanation.